layers/+completion/spacemacs-helm/packages.el

```diff
@@ -185,7 +204,8 @@
         "rs"   'spacemacs/resume-last-search-buffer
         "ry"   'helm-show-kill-ring
         "sl"   'spacemacs/resume-last-search-buffer
-        "sj"   'spacemacs/jump-in-buffer)
+        "sj"   'spacemacs/jump-in-buffer
+        "aR"   'helm-available-repls)
```

Comment:
Man, bad habits putting things at the end. Anyway, seems like that key-binding is taken so I'm removing it until we find a free one :joy: